core/src/stmt.rs

```diff
@@ -9,4 +12,15 @@ pub trait MongoStatement: Debug {
     // Get the BSON value for the cell at the given colIndex on the current row.
     // Fails if the first row has not been retrieved (next must be called at least once before getValue).
     fn get_value(&self, col_index: u16) -> Result<Option<Bson>>;
+    // Return a reference to the ResultSetMetadata for this Statement.
+    fn get_resultset_metadata(&self) -> &Vec<MongoColMetadata>;
+    // get_col_metadata gets the metadata for a given column, 1-indexed as per the ODBC spec.
```

Comment:
This is good! Should `get_value` also be 1-indexed? 🤔

Reply:
Yes!